src/ui/server/index.ts

```diff
@@ -111,18 +104,6 @@ ui.use(
   }),
 );
 
-if (process.env.NODE_ENV !== 'production') {
-  ui.use(
-    basicAuth({
-      users: {
-        // @ts-ignore
-        [process.env.BASIC_AUTH_KEY]: process.env.BASIC_AUTH_SECRET, // @ts-ignore
-      },
-      challenge: true,
-    }),
-  );
-}
-
```

Comment:
Without basic auth, am I right in thinking, anyone could access e.g dev or staging environment?

Reply:
That is correct, however before deployment IaC will be updated to only allow VPN connections.